### PR TITLE
Docs(web-twig): Introduce Button story

### PR DESCRIPTION
**Pull request by literat**
refs #DS-247

### CODE REVIEW

File: packages/web-twig/src/Resources/components/Button/Button.stories.twig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{% extends 'layout/default.html.twig' %}
+
+{% block content %}
+
+{% set colors = [
+    "primary", "secondary", "tertiary", "inverted", "success", "informative", "warning", "danger"
+]%}
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/Button/Button.stories.twig:7]:
```suggestion
] %}
```

File: packages/web-twig/src/Resources/components/Button/Button.stories.twig [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+{% set sizes = [ "small", "medium", "large"] %}
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/Button/Button.stories.twig:8]:
```suggestion
{% set sizes = [ "small", "medium", "large" ] %}
```

File: packages/web-twig/src/Resources/components/Button/stories/ButtonColorsSizes.twig [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  <div class="Grid Grid--cols-1 Grid--desktop--cols-3">
+    {% for size in sizes %}
+    <div>
+      <h3 class="docs-Heading">Size {{ size }}</h3>
+      {% for color in colors %}
+      <div class="{% if color is same as('inverted') %}docs-Box {% endif %}my-600">
+        <div>
+          <Button color={color} size={size} onClick="alert('rimmer')">Button {{ color }}</Button>
+          <Button color={color} size={size}>
+            <svg class="mr-400" width="24" height="24" aria-hidden="true">
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/Button/stories/ButtonColorsSizes.twig:13]: We should use Icon component here

**Review by crishpeen — CHANGES_REQUESTED** [review]

File: packages/web-twig/src/Resources/components/Button/stories/ButtonDisabled.twig [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  <div class="Grid Grid--cols-1 Grid--tablet--cols-2 Grid--desktop--cols-3">
+    {% for size in sizes %}
+    <div>
+      <h3 class="docs-Heading">Size {{size}}</h3>
+      {% for color in colors %}
+      <div class="{% if color is same as('inverted') %}docs-Box {% endif %}my-600">
+        <div>
+          <Button color={color} size={size} isDisabled>Disabled {{ color }}</Button>
+          <Button color={color} size={size} isDisabled>
+            <svg class="mr-400" width="24" height="24" aria-hidden="true">
```
Comment (tomassychra) [inline packages/web-twig/src/Resources/components/Button/stories/ButtonDisabled.twig:13]: As @crishpeen already mentioned, wouldn't the `Icon` be better?

**Review by tomassychra — APPROVED** [review]

File: packages/web-twig/src/Resources/components/Button/Button.stories.twig (hunk 2 above)
Comment (literat) [reply packages/web-twig/src/Resources/components/Button/Button.stories.twig]: Fixed.

File: packages/web-twig/src/Resources/components/Button/stories/ButtonColorsSizes.twig (hunk 3 above)
Comment (literat) [reply packages/web-twig/src/Resources/components/Button/stories/ButtonColorsSizes.twig]: Fixed.

File: packages/web-twig/src/Resources/components/Button/Button.stories.twig (hunk 1 above)
Comment (literat) [reply packages/web-twig/src/Resources/components/Button/Button.stories.twig]: Fixed.

File: packages/web-twig/src/Resources/components/Button/stories/ButtonDisabled.twig (hunk 4 above)
Comment (literat) [reply packages/web-twig/src/Resources/components/Button/stories/ButtonDisabled.twig]: Fixed.

**Review by crishpeen — APPROVED** [review]

**Pull request merged**